README.md

```diff
@@ -115,10 +113,11 @@ var editor = new MediumEditor('.editable', {
     toolbar: {
         /* These are the default options for the toolbar,
            if nothing is passed this is what is used */
-        buttons: ['bold', 'italic', 'underline', 'anchor', 'header1', 'header2', 'quote'],
+        buttons: ['bold', 'italic', 'underline', 'anchor', 'h1', 'h2', 'quote'],
         diffLeft: 0,
         diffTop: -10,
         firstButtonClass: 'medium-editor-button-first',
+        headerTags: ['h2', 'h3', 'h4'],
```

Comment:
The reason why I wouldn't want to put all available header tags is that we want the first header to be a `<h2>` and not a `<h1>`.  Also, it'd be good if the 'H4' button was NOT supported OOB, but was still possible.

To match medium, we need the 'H1' button to produce a `<h2>`, which would mean we wouldn't want to put all the headers in there.  Using `headerTags: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6']` wouldn't match medium since we'd have to start with a `<h1>`.  

By putting all 3 header tags in the array, that means the 'H3' button will work OOB, the user just needs to add 'h3' to the buttons array.  

Also, but only putting these 3 options, an 'H4' button won't work unless the user changes the `headerTags` array somehow.  Based on this reasoning I landed on supplying exactly 3 default `headerTags`...